metadata.rb

```diff
@@ -6,6 +6,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '2.0.2'
 supports         'windows'
+depends          'ohai', '>= 4.0.0'
```

Comment:
i plan on submitting the plugin to the ohai project, so it can be removed in the future.